parm/config/gfs/config.com

```diff
@@ -94,6 +94,12 @@ declare -rx COM_ICE_NETCDF_TMPL=${COM_BASE}'/products/ice/netcdf'
 declare -rx COM_ICE_GRIB_TMPL=${COM_BASE}'/products/ice/grib2'
 declare -rx COM_ICE_GRIB_GRID_TMPL=${COM_ICE_GRIB_TMPL}'/${GRID}'
 
+declare -rx COM_RFCST_PROD_ATMOS_F2D_TMPL=${COM_BASE}'/products/rfcst/atmos/f2d'
+declare -rx COM_RFCST_PROD_ATMOS_F3D_TMPL=${COM_BASE}'/products/rfcst/atmos/f3d'
+declare -rx COM_RFCST_PROD_OCN_TMPL=${COM_BASE}'/products/rfcst/ocn'
+declare -rx COM_RFCST_PROD_ICE_TMPL=${COM_BASE}'/products/rfcst/ice'
+declare -rx COM_RFCST_PROD_WAV_TMPL=${COM_BASE}'/products/rfcst/wav'
+
```

Comment:
If an output file for a given lead time has failed to produce, I do not think that should cause the entire rocoto task to fail, especially since all lead times (and ocean, ice, atmos and wave compoents) are being processed in a single rocoto task.

Reply:
You should probably use `ARCDIR`, which is what the GFS uses. I wouldn't call these `COM`, I'd call them something else  to avoid confusion (and they can be moved to `config.extractvars` as nothing else uses them).

Reply:
Thanks, @WalterKolczynski-NOAA. I am removing these from `config.base` since these are not needed by any other job downstream. The archive job will not need this because these extracted model fields will not be needed on HPSS. I will also remove these from `JGLOBAL_EXTRACTVARS`. I will define an `arcdir `variable in `config.extractvars`.